src/main/kotlin/uk/gov/dluhc/printapi/rest/CertificateController.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.dluhc.printapi.rest
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.CrossOrigin
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.dluhc.printapi.models.CertificateSummaryResponse
+
+@RestController
+@CrossOrigin
+class CertificateController {
+    companion object {
+        const val ERO_VC_ADMIN_GROUP_PREFIX = "ero-vc-admin-"
+    }
+
+    @GetMapping("/eros/{eroId}/certificates/applications/{applicationId}")
+    @PreAuthorize(
+        """
+        hasAnyAuthority(
+            T(uk.gov.dluhc.printapi.rest.CertificateController).ERO_VC_ADMIN_GROUP_PREFIX.concat(#eroId)
+        )
+        """
+    )
+    fun getCertificateSummaryByApplicationId(
```

Comment:
@ResponseStatus(HttpStatus.OK)

Reply:
Thanks, @vishalgupta-valtech , for the suggestions and approval.
I just double checked without the annotation to be sure if the response status is 200 for a successful execution and the answer is yes. 
Since this seems to be the default behavior, I am leaned to omitting the annotation.